k8s/helmfile/env/local/minio.values.yaml.gotmpl

```diff
@@ -13,7 +13,29 @@ persistence:
 buckets:
   - name: backups
     policy: public
+  - name: api-assets
+    policy: public
 
 resources:
   requests:
     memory: 0.5Gi
+
+consoleService:
+  port: "80"
+
+consoleIngress:
```

Comment:
This exposes an admin interface for minio which lets you browse the file tree et. al. It's not strictly necessary but it was helpful for me to get this set up. Not sure if we want to keep it or remove it again. For local, I can see no real harm in leaving it.

Reply:
I can imagine that this will be helpful in the future as well! works nicely